draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -103,49 +96,64 @@ This specification defines the format of the Client Attestation that a Client In
+## Client Attestation HTTP Headers
 
-The value of the "client_assertion_type" parameter (as defined in {{RFC7521}}) set to "urn:ietf:params:oauth:client-assertion-type:jwt-client-attestation".
+A Client Attestation JWT and Client Attestation PoP JWT is included in an HTTP request using the following request header fields.
 
-The value of the "client_assertion" parameter (as defined in {{RFC7521}}) set to a value containing two JWTs, separated by a '~' character. It MUST NOT contain more or less than precisely two JWTs separated by the '~' character. The first JWT MUST be the client attestation JWT defined in [](#client-attestation-jwt), the second JWT MUST be the client attestation PoP defined in [](#client-attestation-pop-jwt).
+OAuth-Client-Attestation:
+:  A JWT that adheres to the structure and syntax as defined in [](#client-attestation-jwt)
 
-The following example demonstrates client authentication using this scheme during the presentation of an authorization code grant in an access token request (with extra line breaks for display purposes only):
+OAuth-Client-Attestation-PoP:
+: A JWT that adheres to the structure and syntax as defined in [](#client-attestation-pop-jwt)
+
+The following is an example of the OAuth-Client-Attestation header.
 
 ~~~
-POST /token HTTP/1.1
-Host: as.example.com
-Content-Type: application/x-www-form-urlencoded
+OAuth-Client-Attestation: eyJhbGciOiAiRVMyNTYiLCJraWQiOiAiMTEifQ.eyJpc3MiOiJodHRwczovL2NsaWVudC5leGFtcGxlLmNvbSIsInN1YiI6Imh0dHBz\
+Oi8vY2xpZW50LmV4YW1wbGUuY29tIiwibmJmIjoxMzAwODE1NzgwLCJleHAiOjEzMDA4MTkzODAsImNuZiI6eyJqd2siOnsia3R5IjoiRUMiLCJ1c2UiOiJzaWciLCJjc\
+nYiOiJQLTI1NiIsIngiOiIxOHdITGVJZ1c5d1ZONlZEMVR4Z3BxeTJMc3pZa01mNko4bmpWQWlidmhNIiwieSI6Ii1WNGRTNFVhTE1nUF80Zlk0ajhpcjdjbDFUWGxGZE\
+FnY3g1NW83VGtjU0EifX19.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+~~~
+
+The following is an example of the OAuth-Client-Attestation-PoP header.
 
-grant_type=authorization_code&
-code=n0esc3NRze7LTCu7iYzS6a5acc3f0ogp4&
-client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3A
-client-assertion-type%3Ajwt-client-attestation&
-client_assertion=eyJhbGciOiJSUzI1NiIsImtpZCI6IjIyIn0.
-eyJpc3Mi[...omitted for brevity...].
-cC4hiUPo[...omitted for brevity...]~eyJzI1NiIsImtphbGciOimtpZCI6IjIyIn0.
-IjIyIn0[...omitted for brevity...].
-iOiJSUzI1[...omitted for brevity...]
+~~~
+OAuth-Client-Attestation-PoP: eyJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJodHRwczovL2NsaWVudC5leGFtcGxlLmNvbSIsImF1ZCI6Imh0dHBzOi8vYXMuZXhhbX\
+BsZS5jb20iLCJuYmYiOjEzMDA4MTU3ODAsImV4cCI6MTMwMDgxOTM4MH0.coB_mtdXwvi9RxSMzbIey8GVVQLv9qQrBUqmc1qj9Bs
 ~~~
 
-## JWT Format and Processing Requirements {#jwt-format-and-processing-requirements}
+Note that per {{RFC9110}} header field names are case-insensitive; so OAUTH-CLIENT-ATTESTATION, oauth-client-attestation, etc., are all valid and equivalent
+header field names. Case is significant in the header field value, however.
```

Comment:
```suggestion
As specified in {{RFC9110}}, header field names are case-insensitive; therefore, OAUTH-CLIENT-ATTESTATION, oauth-client-attestation, and other case variations are all valid and equivalent header field names. However, it is important to consider that the case is significant in the value of the header field.
```

Reply:
I think the current language is sufficient even though it is a little more brief.